### ISSUE TITLE
Project Restructure and handover to the SAP Open Source Program Office

### ISSUE THREAD

**Issue by Shegox** [cla-assistant/cla-assistant#735]
This is a general information issue for whoever it might concerns.

The TL;DR is that the project ownership of CLA-assistants transitions SAP internally from our developer tooling team to the Open Source Program Office. The general functionality will keep running, but we will do a bit project/repository/organization cleanup.

Something which will happen are following: 
- [x] Removing all contributors except the "core" team from the organization. Nowadays it is possible to assign issues to external contributors as well, meaning that even for collaboration an organization membership is not needed and this helps us to keep a better overview about who is part of the core. 
(affected @yaseminc, @tim-mc, @smorimoto, @mikheevm, @MichaelTsengLZ, @jsoref, @janczizikow, @chrisgavin, @BenLloydPearson, @AnandChowdhary, @amateurhuman, @datenreisender )
- [x] Remove some of the GitHub Apps, which are no longer needed.
- [x] Update repository structure #731
- [ ] Cleanup current open issues
- [More later]

**Comment by Shegox** [comment]
@jsoref thanks for pointing that out :)

And everyone is still more than welcome to help out with issues or answering questions 👍

**Comment by MichaelTsengLZ** [comment]
Thanks for the notice.

From: Tobias ***@***.***>
Date: Wednesday, August 11, 2021 at 12:06 AM
To: cla-assistant/cla-assistant ***@***.***>
Cc: Michael Zeng (OSE) ***@***.***>, Mention ***@***.***>
Subject: [cla-assistant/cla-assistant] Project Restructure and handover to the SAP Open Source Programm Office (#735)

This is a general information issue for whoever it might concerns.

The TL;DR is that the project ownership of CLA-assistants transitions SAP internally from our developer tooling team to the Open Source Program Office. The general functionality will keep running, but we will do a bit project/repository/organization cleanup.

Something which will happen are following:

  *   Removing all contributors except the "core" team from the organization. Nowadays it is possible to assign issues to external contributors as well, meaning that even for collaboration an organization membership is not needed and this helps us to keep a better overview about who is part of the core.
(affected @yaseminc<https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fyaseminc&data=04%7C01%7Cmiczeng%40microsoft.com%7C5385dfccf52b411c7a2408d95c968fee%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C637642623999722833%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C1000&sdata=reYsFv4DA%2FF%2Fnwnv3FNnTT9F3vfp%2Fh0djAHWjG6CyvA%3D&reserved=0>, @tim-mc<https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Ftim-mc&data=04%7C01%7Cmiczeng%40microsoft.com%7C5385dfccf52b411c7a2408d95c968fee%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C637642623999722833%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C1000&sdata=ZUHtIXVD7hcU9yuS%2B3XUiOA8mbnBBJ1Uay7SaJ6bcJs%3D&reserved=0>, @smorimoto<https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fsmorimoto&data=04%7C01%7Cmiczeng%40microsoft.com%7C5385dfccf52b411c7a2408d95c968fee%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C637642623999732823%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C1000&sdata=Aw3TQ76BJN8SSmmf0RUFrWVkXcqVvAsj8W9ylR10WCA%3D&reserved=0>, @mikheevm<https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fmikheevm&data=04%7C01%7Cmiczeng%40microsoft.com%7C5385dfccf52b411c7a2408d95c968fee%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C637642623999742823%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C1000&sdata=5TcsrWQ5zq%2BCYpG%2Bv%2Bhez61nOwa8TAFmUaBOErpU9Wo%3D&reserved=0>, @MichaelTsengLZ<https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2FMichaelTsengLZ&data=04%7C01%7Cmiczeng%40microsoft.com%7C5385dfccf52b411c7a2408d95c968fee%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C637642623999752815%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C1000&sdata=jK28aI7k1SN2o4lVfSMSufmTAp7WCwfzqpmtyY6OYSs%3D&reserved=0>, @jsoref<https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fjsoref&data=04%7C01%7Cmiczeng%40microsoft.com%7C5385dfccf52b411c7a2408d95c968fee%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C637642623999752815%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C1000&sdata=AR9J9aMW7pJyCGoDNU5LqwECyXOctq15ZaZzy84tSCA%3D&reserved=0>, @janczizikow<https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fjanczizikow&data=04%7C01%7Cmiczeng%40microsoft.com%7C5385dfccf52b411c7a2408d95c968fee%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C637642623999762809%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C1000&sdata=4X%2BNLrVmIwCMOC8zJn9L9KAbpx9pgRL7NXCJljhSc%2Bo%3D&reserved=0>, @chrisgavin<https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fchrisgavin&data=04%7C01%7Cmiczeng%40microsoft.com%7C5385dfccf52b411c7a2408d95c968fee%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C637642623999772804%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C1000&sdata=zIliP%2BBxUuaFyT7s8ovitijarz38xoLoIcEI22wRFBY%3D&reserved=0>, @BenLloydPearson<https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2FBenLloydPearson&data=04%7C01%7Cmiczeng%40microsoft.com%7C5385dfccf52b411c7a2408d95c968fee%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C637642623999782796%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C1000&sdata=re%2FEzSCyr1XV2tPgThyL%2BydtA1VchtNsb6DThXDgO1o%3D&reserved=0>, @AnandChowdhary<https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2FAnandChowdhary&data=04%7C01%7Cmiczeng%40microsoft.com%7C5385dfccf52b411c7a2408d95c968fee%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C637642623999782796%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C1000&sdata=S3IxtsSKWJbkKaS8TgziRn9kc7GrGinj1mu3x4i%2F3GE%3D&reserved=0>, @amateurhuman<https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Famateurhuman&data=04%7C01%7Cmiczeng%40microsoft.com%7C5385dfccf52b411c7a2408d95c968fee%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C637642623999792795%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C1000&sdata=YvkAIOUg4m5qEI9EPq3QqhhOPr5KDZFMYajS3a0T51A%3D&reserved=0>)
  *   Remove some of the GitHub Apps, which are no longer needed.
  *   Update repository structure #731<https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fcla-assistant%2Fcla-assistant%2Fpull%2F731&data=04%7C01%7Cmiczeng%40microsoft.com%7C5385dfccf52b411c7a2408d95c968fee%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C637642623999802786%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C1000&sdata=mTfojuIMSIotJ1xgMvPTI4%2B6W8J4sBiYvV45cbUkMy4%3D&reserved=0>
  *   [More later]

—
You are receiving this because you were mentioned.
Reply to this email directly, view it on GitHub<https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fcla-assistant%2Fcla-assistant%2Fissues%2F735&data=04%7C01%7Cmiczeng%40microsoft.com%7C5385dfccf52b411c7a2408d95c968fee%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C637642623999802786%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C1000&sdata=A0bTiMyHkkvP2JRrG388rIPRtIL5O07s%2Faw%2BZZiWFeY%3D&reserved=0>, or unsubscribe<https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fnotifications%2Funsubscribe-auth%2FAE3WSL2KPTZZHUTBOTAOGN3T4IOP3ANCNFSM5B5VVFBQ&data=04%7C01%7Cmiczeng%40microsoft.com%7C5385dfccf52b411c7a2408d95c968fee%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C637642623999812774%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C1000&sdata=NvcuBkgROMBIiYnFaCVx0cxGTT0fZ3bAnpN2V%2Fx464c%3D&reserved=0>.
Triage notifications on the go with GitHub Mobile for iOS<https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Fapps.apple.com%2Fapp%2Fapple-store%2Fid1477376905%3Fct%3Dnotification-email%26mt%3D8%26pt%3D524675&data=04%7C01%7Cmiczeng%40microsoft.com%7C5385dfccf52b411c7a2408d95c968fee%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C637642623999812774%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C1000&sdata=npw4kcgmP2mr8cOYOfjHrtiMw0LNlZ2L0wLrqASYY7A%3D&reserved=0> or Android<https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Fplay.google.com%2Fstore%2Fapps%2Fdetails%3Fid%3Dcom.github.android%26utm_campaign%3Dnotification-email&data=04%7C01%7Cmiczeng%40microsoft.com%7C5385dfccf52b411c7a2408d95c968fee%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C637642623999822767%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C1000&sdata=27RK6WCSbbXXnhsUcepCgE6niI8KXn0Ggs%2FNAj%2BK95Y%3D&reserved=0>.
